src/app/components/JsonSchemaConfigurator/CheckboxWithDescriptionField.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { connectField, filterDOMProps } from 'uniforms';
+
+import { Checkbox } from '@patternfly/react-core';
+
+export type CheckboxWithDescriptionProps = {
+  id: string;
+  disabled: boolean;
+  error: boolean;
+  name: string;
+  description: string;
+  value: boolean;
+  onChange: (value: boolean) => void;
+  [key: string]: any;
+};
+function CheckboxWithDescription({
+  id,
+  label,
+  type,
+  disabled,
+  error,
+  errorMessage,
+  showInlineError,
+  help,
+  description,
+  value,
+  onChange,
+  required,
+  ...props
+}: CheckboxWithDescriptionProps) {
+  return (
+    <Checkbox
```

Comment:
I think we still need to wrap this in a FormGroup [like this example](https://patternfly-react-main.surge.sh/components/form#basic), to be consistent with the other form controls.  You can pass all the same properties [as here](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/app/components/JsonSchemaConfigurator/TypeaheadField.tsx#L36-L45).  I guess if we add another custom component we may consider a wrapper function that injects this, or maybe we could go and expose the wrapper function that uniforms-patternfly uses for this.

Reply:
Sure I will add the "FormGroup", But I am not sure if we need all the same properties here as in [Typeahead](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/app/components/JsonSchemaConfigurator/TypeaheadField.tsx#L36-L45) i.e label, type, isRequired may not be applicable here